contracts/libraries/NFTDescriptor.sol

```diff
@@ -70,15 +73,28 @@ library NFTDescriptor {
         string memory description =
             string(
                 abi.encodePacked(
-                    'Represents a liquidity position in a Uniswap V3 pool. Redeemable for owed reserve tokens.',
-                    '\\nliquidity: ',
-                    uint256(params.liquidity).toString(),
-                    '\\npoolAddress: ',
+                    'This NFT represents a liquidity position in a Uniswap V3 ',
+                    token1Symbol,
```

Comment:
do we need to do ordering of these symbols the same way we do prices? otherwise the prices may be confused to be flipped

Reply:
e.g. we show ETH/DAI and then prices 1500-2500

Reply:
I've attempted to order them consistently throughout. Am I missing something?
(see line 44)

Reply:
ah gotcha, yeah better to name them quoteToken and baseToken or something like that